cluster-autoscaler/cloudprovider/gce/mig_target_sizes_provider.go

```diff
@@ -55,8 +55,10 @@ func (c *cachingMigTargetSizesProvider) GetMigTargetSize(migRef GceRef) (int64,
 	}
 
 	newTargetSizes, err := c.fillInMigTargetSizeCache()
-	if err != nil {
-		// fallback to querying for single mig
+
+	// if we still do not have value here return an error
```

Comment:
that is not what this code does, the previous comment described the actual logic better (ie. you fallback to FetchMigTargetSize and only return an error if you fail it)

Reply:
Changed to the previous comment